server/push-script.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+#use TAG env. variable to create the container with the given tag
+TAG="${TAG:-latest}"
```

Comment:
Could it be both latest and actual tag?

Reply:
It works this way - when something is published to `master` it makes it without tag, so it's published as `latest`, on release it provides the `TAG`.
It can happen that the `latest` and latest released version are not the same. I'm not sure, how to fix it. 
Maybe build it only on release and make it publish both version with the tag and `latest`.